code/script.js

```diff
@@ -4,16 +4,127 @@
+
+  // Step 3 - Subtype choice
+  pastaSubtype = prompt("Please select a pasta:\n1. Spaghetti\n2. Fettuccine Alfredo\n3. Pasta Agli Oglio");
+
+  if (pastaSubtype === "1") {
+    alert("You have selected Spaghetti Pasta!");
+  } else if (pastaSubtype === "2") {
+    alert("You have selected Fettuccine Alfredo Pasta!");
+  } else if (pastaSubtype === "3") {
+    alert("You have selected Pasta Agli Oglio!");
+  } else {
+    alert("Invalid choice. Please select a valid subtype.");
+  }
+
+} else if (selectedOption === 3) {
+  alert("You have selected Salad!");
+
+  // Step 3 - Subtype choice
+  saladSubtype = prompt("Please select a salad:\n1. Caesar Salad\n2. Greek Salad\n3. Garden Salad");
+
+  if (saladSubtype === "1") {
+    alert("You have selected Caesar Salad!");
+  } else if (saladSubtype === "2") {
+    alert("You have selected Greek Salad!");
+  } else if (saladSubtype === "3") {
+    alert("You have selected Garden Salad!");
+  } else {
+    alert("Invalid choice. Please select a valid subtype.");
+  }
+} else {
+  alert("Invalid choice. Please select a valid option.");
+}
 
 // Step 4 - Age
-// Your code goes here
+let userInputAge = prompt("Is this meal for a child or an adult? Please enter your age");
+let age = parseInt(userInputAge);
 
-// Step 5 - Order confirmation
-// Your code goes here
+if (!isNaN(age)) {
+  let ageGroup = (age < 11) ? "child" : "adult";
+
+  // Step 5 - Order confirmation
+  let foodTypeText;
+  let foodPrice;
+
+  if (selectedOption === 1) {
+    if (pizzaSubtype === "1") {
+      foodTypeText = "Margherita Pizza";
+    } else if (pizzaSubtype === "2") {
+      foodTypeText = "Pepperoni Pizza";
+    } else if (pizzaSubtype === "3") {
+      foodTypeText = "Veggie Pizza";
+    }
+  } else if (selectedOption === 2) {
+    if (pastaSubtype === "1") {
+      foodTypeText = "Spaghetti Pasta";
+    } else if (pastaSubtype === "2") {
+      foodTypeText = "Fettuccine Alfredo Pasta";
+    } else if (pastaSubtype === "3") {
+      foodTypeText = "Pasta Agli Oglio";
+    }
+  } else if (selectedOption === 3) {
+    if (saladSubtype === "1") {
+      foodTypeText = "Caesar Salad";
+    } else if (saladSubtype === "2") {
+      foodTypeText = "Greek Salad";
+    } else if (saladSubtype === "3") {
+      foodTypeText = "Garden Salad";
+    }
+  }
+
+  if (ageGroup === "child") {
+    foodPrice = 7;  // Child's price
+  } else {
+    foodPrice = 12; // Adult's price
+  }
+
+  let confirmationMessage = `You have chosen a ${ageGroup} ${foodTypeText} for $${foodPrice}. Would you like to confirm your order? \n1. Yes \n2. No`;
+
+  let confirmation = prompt(confirmationMessage);
+
+  if (confirmation === "1") {
+    alert("Thank you for choosing Javascript Pizzeria! Your order is now being made. Enjoy!");
+  } else if (confirmation === "2") {
+    alert("Thank you for considering us. Feel free to return for future orders!");
+  }
+
+} else {
+  alert("Invalid input. Please enter a valid age.");
```

Comment:
i think you should alert please enter a valid number 1 or2 not age because your prompt here just confirm yes or no